about/billing.html.markerb

```diff
@@ -42,22 +48,59 @@ Stopped Machines are billed based on their root file system (rootfs) usage per s
 
 For example, a Machine described in your dashboard as having 1GB of rootfs stopped for 10 days in the entire month will cost $0.05. If you have 3 stopped Machines of 1GB rootfs each stopped for 30 days it will cost $0.45 ($0.15 x 3).
 
+---
+
 ## GPU billing
 
 GPUs are billed per second that the attached Fly Machine is running (the time they spend in the `started` state), based on the per hour cost of the GPU card. See [pricing for GPUs](/docs/about/pricing/#gpus-and-fly-machines).
 
 You're also billed for the Fly Machine separately from the GPU.
 
+---
+
 ## Volume billing
 
 Volume billing is pro-rated to the hour and we subtract the free allowances first for the Launch, Scale, and Enterprise plans. For details, see [Volume pricing](/docs/about/pricing/#persistent-storage-volumes).
 
 If you create a volume, you will be charged for it. You’re billed for volumes that aren’t attached to Machines, and for volumes that are attached to Machines in any state, including stopped Machines.
 
+---
+
+## Unified Billing
+
+Unified billing allows you to consolidate billing for multiple organizations under a single organization. This means  You'll receive a single invoice for all organizations under unified billing. This feature is useful for managing billing for multiple organizations, such as when you have multiple teams or projects.
+
+**Billing Organization**: a normal organization with a payment method linked to it. This organization is responsible for paying for the resources used by all organizations linked to it.
+
+**Linked Organization**: an organization that has its billing linked to a another organization. It otherwise functions a normal organization
+
+### How Unified Billing Works
+
+- You'll receive a single invoice for all organizations under unified billing
+- Billing Organization must have a payment method on file (credit card or Stripe Link)
+- Linked Organizations inherit the plan of their Billing Organization
```

Comment:
> - Linked Organizations inherit the plan of their Billing Organization

this might raise questions. can we clarify what it means exactly?

Reply:
I actually think it might be more-than-necessary information. we can clarify this detail if customers start to ask, but it's probably a distinction that only matters to those of us making the platform.